frontend/public/components/overview/namespace-overview.tsx

```diff
@@ -57,37 +57,36 @@ const QuotaBox = ({quota, resourceTypes}) => {
         </h2>
       </div>
       <div className="container-fluid group__graphs">
-        <QuotaGaugeCharts quota={quota} resourceTypes={resourceTypes} rowClass="co-resource-quota-chart-row--compact"
-          chartClass="graph-wrapper--compact" />
+        <QuotaGaugeCharts quota={quota} resourceTypes={resourceTypes} compact />
         {scopes && <dl className="co-m-pane__details quota-dashboard-scopes">
-          <QuotaScopes scopes={scopes} compact={true} />
+          <QuotaScopes scopes={scopes} compact />
         </dl>}
       </div>
     </div>
   </div>;
 };
 
-class QuotaBoxes extends React.PureComponent<QuotaBoxesProps> {
-  render() {
-    const { loaded, loadError, data: quotas } = this.props.resourceQuotas;
+const QuotaBoxes: React.SFC<QuotaBoxesProps> = ({resourceQuotas}) => {
+  const { loaded, loadError, data: quotas } = resourceQuotas;
 
-    if (!loaded) {
-      return <LoadingInline />;
-    }
-    if (loadError) {
-      <StatusBox loadError={loadError} />;
-    }
-    const quotaBoxes = _.reduce(quotas, (accumulator, quota) => {
-      const resourceTypes = getQuotaResourceTypes(quota);
-      if (hasComputeResources(resourceTypes)) {
-        accumulator.push(<QuotaBox quota={quota} resourceTypes={resourceTypes} />);
-      }
-      return accumulator;
-    }, []);
-    const rows = _.chunk(quotaBoxes, 2);
-    return _.map(rows, (row, index) => <div key={index} className="row quota-dashboard-row">{row}</div>);
+  if (!loaded) {
+    return <LoadingInline />;
+  }
+  if (loadError) {
+    <StatusBox loadError={loadError} />;
   }
-}
+  const quotaBoxes = _.reduce(quotas, (accumulator, quota: K8sResourceKind) => {
+    const resourceTypes = getQuotaResourceTypes(quota);
+    if (hasComputeResources(resourceTypes)) {
+      accumulator.push(<QuotaBox key={quota.metadata.name} quota={quota} resourceTypes={resourceTypes} />);
+    }
+    return accumulator;
+  }, []);
+  const rows = _.chunk(quotaBoxes, 2);
+  return <React.Fragment>
+    {_.map(rows, (row, index) => <div key={index} className="row quota-dashboard-row">{row}</div>)}
+  </React.Fragment>;
```

Comment:
I don't think React.Fragment is needed? Does this work?

```jsx
return _.map(rows, (row, index) => <div key={index} className="row quota-dashboard-row">{row}</div>)l
```

Reply:
I believe this is what I had before and was getting Typescript errors.